examples/.env.example

```diff
@@ -0,0 +1,6 @@
+SPEECH_TO_TEXT_IAM_APIKEY=""
```

Comment:
Can you remove the `""`?

Reply:
It works with "" perfectly but what do you say that instead I will "<your IAM API key>" and "<your API username>" etc. for all fields?

Reply:
My problem is that we have 2 set of credentials so it's unclear for users which one they should complete as they all have `""`.

Maybe doing something like

```properties
TEXT_TO_SPEECH_IAM_APIKEY=<Your API key>
SPEECH_TO_TEXT_URL=https://stream.watsonplatform.net/speech-to-text/api

# If your Speech to Text credentials contain username and password use
#SPEECH_TO_TEXT_USERNAME=
#SPEECH_TO_TEXT_PASSWORD=

TEXT_TO_SPEECH_IAM_APIKEY=<Your API key>
TEXT_TO_SPEECH_URL=https://stream.watsonplatform.net/text-to-speech/api

# If your Text to Speech credentials contain username and password use
#TEXT_TO_SPEECH_USERNAME=""
#TEXT_TO_SPEECH_PASSWORD="" 
```

I'm adding the URL field because right now the example will only work in `US-South` but the service is available in other regions.

Reply:
I wrote in the readme that you should *only* set the IAM API key if you're using it but I agree with your comment. Regarding the URL field, I was about to open a case since I did this work already on my end since the code didn't support my Tokyo & Sydney based services. I will contact you to share my analysis and work regarding this outside of this issue but I will also add the URL for both services now.